plugins/bruce.py

```diff
@@ -0,0 +1,13 @@
+import requests
+
+from will.plugin import WillPlugin
+from will.decorators import respond_to
+
+class CryptoPlugin(WillPlugin):
+    @respond_to("crypto")
```

Comment:
this matches just the word by itself on a line as opposed to matching on "cryptohackers" or "scyrpto"